### DISCUSSION TITLE
AI on MDN: money involved?

### DISCUSSION THREAD

**Discussion by SphinxKnight** [mdn/mdn-community#417] [Announcements]
When there was a "discussion" about ads, it was clear that advertisement was a way for Mozilla Corporation to have revenue from the MDN ""product"".
From several years now, revenue-making has been the compass of big "product" investments (MDN Plus, then Ads). 

Can Mozilla Corporation state clearly if there was a financial deal in the case of AI integration on MDN?
If that was not the case, how was AI addition prioritized? What was the incentive to have such features on MDN (apart from hype training)? (see https://github.com/orgs/mdn/discussions/409)

**Comment by caugner** [comment]
We have recorded your question as number 30.

**Comment by Rumyra** [comment]
Hi @SphinxKnight - there is no financial agreement between Mozilla and OpenAI. We pay for using OpenAI’s API.

**Reply by SphinxKnight** [reply]
Thank you for the reply @Rumyra

**Reply by nicuveo** [reply]
@Rumyra the phrasing of your answer makes it clear that there is no financial agreement between Mozilla and OpenAI, but that doesn't fully answer the question: can you confirm that there isn't *any* financial incentive for Mozilla to integrate LLMs?

**Reply by SphinxKnight** [reply]
To be honest, I think the question I asked was clearly answered (deal/agreement != incentive (at least how I intended it)).
I'll let anyone from Mozilla add anything if they want to but I'd prefer to keep Ruth's answer for this (in terms of GitHub Discussion mgmt/UI)
